schema/playground/player_info.schema

```diff
@@ -0,0 +1,28 @@
+package playground;
+
+component PlayerInfo {
+   id = 2000;
+ 
+   int32 player_id = 1;
+}
+ 
+component PlayerControls {
```

Comment:
Is this not already a component?

Reply:
Branched off master which does not have this as far as I can tell